src/consensus/raft/raft.h

```diff
@@ -969,8 +971,8 @@ namespace raft
 
     void restart_election_timeout()
     {
-      // Randomise timeout_elapsed to get an election timeout that is
-      // effectively between zero and double the configured election timeout.
+      // Randomise timeout_elapsed to get a random election timeout
+      // between 0.5x and 1x the configured election timeout.
```

Comment:
I think the old behaviour was odd and this is more intuitive. If I say the election timeout is 10s, I want an election to happen after 10s of inactivity (maybe sooner for random distribution, sure!), I shouldn't need to know this implementation detail to understand my 20s of downtime. In particular, this means that "wait for 2 full election timeouts" is `2 * election_timeout` rather than `f * election_timeout`.